keepassxc-browser/popups/popup.css

```diff
@@ -1,51 +1,37 @@
 body {
-    font-family: sans-serif;
     overflow-x: hidden;
     background-color: #eee;
```

Comment:
Maybe the popup background color should be white, just like the Settings page.

Reply:
Yeah, agreed

Reply:
Nice. Everything else is already looking much better!